code/modules/mining/machine_vending.dm

```diff
@@ -134,18 +134,15 @@
 		new /datum/data/vendor_equipment("30 Marker Beacons",			/obj/item/stack/marker_beacon/thirty,								150),
 		new /datum/data/vendor_equipment("Shelter Capsule",				/obj/item/survivalcapsule,											400),
 		new /datum/data/vendor_equipment("Regen. Core Stabilizer",		/obj/item/hivelordstabilizer,										400),
-		new /datum/data/vendor_equipment("Skeleton Key",				/obj/item/skeleton_key,												750),
-		new /datum/data/vendor_equipment("Survival Medipen",			/obj/item/reagent_containers/hypospray/medipen/survival,			500),
-		new /datum/data/vendor_equipment("Brute Healing Kit",			/obj/item/storage/firstaid/brute,									600),
-		new /datum/data/vendor_equipment("Burn Healing Kit",			/obj/item/storage/firstaid/fire,									600),
-		new /datum/data/vendor_equipment("Advanced Healing Kit",		/obj/item/storage/firstaid/advanced,								1200),
```

Comment:
I'm scared that besides explorers, miners will also head to medbay to raid medkits with this gone.

Reply:
I don't appreciate the survival pen being highlighted with the others as if it is removed. The survival pen is still present, just with an adjusted cost. 

Again:
> Any role being 100% self-reliant is bad.

Injured players should rely on medbay for treatment.